Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -8,6 +8,7 @@ import Shared
 import XCGLogger
 
 private let log = Logger.browserLogger
+private let desktopBookmarksLabel = NSLocalizedString("Desktop Bookmarks", tableName: "BookmarkPanel", comment: "The folder name for the virtual folder that contains all desktop bookmarks.")
```

Comment:
Yeah, we (and I think the rest of the Swift community?!) dropped that convention, leaving capitals for classes/structs/enums. This came up in conversation on Wednesday with @jhugman and @fluffyemily, coincidentally.

(I didn't want to fix the only other use in this file just yet.)